src/components/CaseEditor.scss

```diff
@@ -79,6 +79,30 @@
   }
 }
 
+div[role=presentation] {
+  display: block;
+}
+
+div[role=menu]{
```

Comment:
We should only use !important as a last ditch effort. There's a lot of !important here. Is there really no other way?